libs/island-ui/core/src/lib/FormStepper/FormStepper.tsx

```diff
@@ -60,20 +60,26 @@ export const FormStepper: FC<{
           [styles.listWithHead]: hasHead,
         })}
       >
-        {sections.map((section, index) => (
-          <FormStepperSection
-            theme={theme}
-            key={`${section.name}-${index}`}
-            section={section}
-            subSection={subSection}
-            isActive={index === activeSection}
-            isComplete={index < activeSection}
-            isLastSection={index === sections.length - 1}
-            sectionIndex={index}
-            activeSubSection={activeSubSection}
-            showSubSectionIcon={showSubSectionIcons}
-          />
-        ))}
+        {sections.map((section, index) => {
```

Comment:
You could also do:

```suggestion
        {sections.filter(section => section?.name !== '').map((section, index) => {
```

Reply:
Nice, I like it thanks. Should be updated now.